route/consistent_hashing.go

```diff
@@ -85,6 +86,18 @@ func (h *ConsistentHasher) AddDestination(d *dest.Destination) {
 		keyBuf.WriteString(":")
 		keyBuf.WriteString(strconv.Itoa(i))
 		position := computeRingPosition(keyBuf.Bytes())
+		//Align with https://github.com/graphite-project/carbon/commit/024f9e67ca47619438951c59154c0dec0b0518c7#diff-1486787206e06af358b8d935577e76f5
+	outer:
```

Comment:
```suggestion
	OUTER:
```

Reply:
seems like the go best practice is camelcased label names
https://golang.org/doc/effective_go.html#switch